addons/web/static/src/js/views/kanban/kanban_view.js

```diff
@@ -70,16 +71,17 @@ var KanbanView = BasicView.extend({
             editable: activeActions.edit,
             deletable: activeActions.delete,
             read_only_mode: params.readOnlyMode,
+            selectionMode: params.selectionMode,
         };
         this.rendererParams.quickCreateEnabled = this._isQuickCreateEnabled();
-        this.rendererParams.readOnlyMode = params.readOnlyMode;
```

Comment:
It seems that readOnlyMode is used in this controller. If we delete this line, the value will be always undefined or false?

Reply:
@adr-odoo It seems that something messed while rebase